test/src/test/java/jenkins/security/StackTraceSuppressionTest.java

```diff
@@ -50,8 +50,8 @@
 
     @Before
     public void setup() {
-        Jenkins.SHOW_STACK_TRACE = false;
-        HttpResponses.SHOW_STACK_TRACE = false;
+        System.setProperty("jenkins.model.Jenkins.SHOW_STACK_TRACE", "false");
```

Comment:
should also be cleared in a tear down `@After` (possibly static) - incase the last test to run sets it to true and we reuse surefire for other tests (which is the default and I did not see changed anywhere in the pom - but also when running multiple tests from the IDE).  

```suggestion
        System.clearProperty("jenkins.model.Jenkins.SHOW_STACK_TRACE");
```